### PR TITLE
fix-593 fixed direction language bug

### PR DESCRIPTION
**Pull request by eidan66**
Added `useState` for re-render the component.

The `useState` ( ```   const [lanChanged, setLanChanged] = React.useState(false);  ```) get boolean value, and use only to set `!lanChanged` when change language.

Now :
when we change language, the state will re-render our app and change the direction.

### CODE REVIEW

File: src/pages/HomePage.tsx [hunk 1]
```diff
@@ -46,10 +47,11 @@ const HomePage: FC<IProps & RouteComponentProps<IRouteProps>> = ({ match }) => {
   useEffect(() => {
     if (match.params.lng) {
       store.changeLanguage(match.params.lng);
+      setLanChanged(!lanChanged);
```
Comment (almogbhl) [inline src/pages/HomePage.tsx:50]: setLanChanged(prevLanChanged => !prevLanChanged);

File: src/pages/HomePage.tsx [hunk 2]
```diff
@@ -46,10 +47,11 @@ const HomePage: FC<IProps & RouteComponentProps<IRouteProps>> = ({ match }) => {
...
     } else {
       store.changeLanguage('he');
     }
-  }, [match.params.lng, store]);
+  }, [lanChanged, match.params.lng, store]);
```
Comment (almogbhl) [inline src/pages/HomePage.tsx:54]: you will always re-render

File: src/pages/HomePage.tsx (hunk 1 above)
Comment (eidan66) [reply src/pages/HomePage.tsx:50]: I still need to see where the problem happened.
My solution need to refix 😉 

Thanks !

**Comment by SejoB** [comment]
will be closed by #676

**Pull request closed without merging**